src/main/java/com/arpnetworking/http/Routes.java

```diff
@@ -135,13 +149,25 @@ public Routes(
                                                         "{\"status\":\""
```

Comment:
Nothing to do with you but can you add a TODO here to not build JSON using strings.